### PR TITLE
Move RefreshSelectorMenuBuilder and RefreshMenuBuilder from jbpm-cons…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…ole-ng to uberfire-extentions to allow reuse across modules

### CODE REVIEW

**Comment by psiroky** [comment]
Could someone review this one? @manstis, @ederign, @Rikkola

**Comment by manstis** [comment]
+1 to merge.

File: uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/menu/RefreshMenuBuilder.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    protected Button menuRefreshButton = GWT.create(Button.class);
+
+    public RefreshMenuBuilder(final SupportsRefresh supportsRefresh) {
+        this.supportsRefresh = supportsRefresh;
+        setupMenuButton();
+    }
+
+    @Override
+    public void push(MenuFactory.CustomMenuBuilder element) {
```
Comment (ederign) [inline uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/menu/RefreshMenuBuilder.java]: is this empty intentionally?

File: uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/menu/RefreshSelectorMenuBuilder.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            }
+        });
+
+        return oneMinuteRadioButton;
+    }
+
+    public void loadOptions(int configuredSeconds) {
+        final AnchorListItem resetButton = new AnchorListItem(constants.Disable_autorefresh());
+
+        if (configuredSeconds > 10) {
```
Comment (ederign) [inline uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/menu/RefreshSelectorMenuBuilder.java]: optional: break this long method in small ones

**Comment by ederign** [comment]
+1.

**Comment by psiroky** [comment]
Merged into 0.8.x

**Pull request closed without merging**